core/src/main/java/org/apache/iceberg/CatalogUtil.java

```diff
@@ -169,6 +175,26 @@ public static Catalog loadCatalog(
     return catalog;
   }
 
+  public static Catalog buildIcebergCatalog(String name, Map<String, String> options, Configuration conf) {
+
```

Comment:
No need to start methods with a newline.

Reply:
done